api/v1/csiscaleoperator_types.go

```diff
@@ -106,6 +106,9 @@ type CSIScaleOperatorSpec struct {
 	// +kubebuilder:default:=/var/lib/kubelet
 	// kubeletRootDirPath is the path for kubelet root directory.
 	KubeletRootDirPath string `json:"kubeletRootDirPath,omitempty"`
+
+	// PodSecurityPolicy name for CSI driver and sidecar pods
+	CSIpspname string `json:"csipspname,omitempty"`
```

Comment:
Can we rename the spec in CR to something like `CsiPSPName`? like in line 108 for `kubeletRootDirPath`. The reason being, when user describes the custom resource the `csipspname` would look like `Csipspname`, but `kubeletRootDirPath` looks like `Kublet Root Dir Path` which has better readability.

Reply:
I have used the name to be consistent with current CNSA PSP variable: https://github.ibm.com/IBMSpectrumScale/scale-core/blob/master/apis/csi/v1/csi_types.go#L108